api/src/pcapi/routes/pro/collective_offers.py

```diff
@@ -318,6 +318,12 @@ def edit_collective_offer_template(
 def patch_all_collective_offers_active_status(
     body: collective_offers_serialize.PatchAllCollectiveOffersActiveStatusBodyModel,
 ) -> None:
+    if body.is_active == True:
```

Comment:
(détail / style) `is_active` est un booléen non nullable, on peut écrire ça plus simplement :

```suggestion
    if body.is_active:
```

idem dans les autres routes